bip-0039.mediawiki

```diff
@@ -111,9 +90,17 @@ The described method also provides plausible deniability, because every passphra
 generates a valid seed (and thus a deterministic wallet) but only the correct one
 will make the desired wallet available.
 
-==Wordlists==
+==Word lists==
+
+[[https://github.com/p2w34/wlips|Creation of word lists moved to a separate repo]].
+There are multiple reasons behind it. The most important ones are:
+* word lists are not Bitcoin-specific
```

Comment:
They are used by Bitcoin, so this doesn't make sense.

Altcoins copying Bitcoin isn't a reason to exclude something from BIPs.

Reply:
I remove this argument